app/models/payment_confirmation.rb

```diff
@@ -23,12 +23,22 @@ def ingest
 
       payroll_run.update!(confirmation_report_uploaded_by: admin_user_id)
 
-      errors.empty? || (raise ActiveRecord::Rollback)
+      if errors.empty?
+        payroll_run.claims.each do |claim|
+          ClaimMailer.payment_confirmation(claim, payment_date.to_i).deliver_later
+        end
+      else
+        raise ActiveRecord::Rollback
+      end
     end
   end
 
   private
 
+  def payment_date
```

Comment:
This method is called `payment_date` but it returns a time object. Maybe things would be better encapsulated if it was rewritten to be called `payment_date_timestamp` instead and included the `to_i` call?